docs/guides/4-custom-rulesets.md

```diff
@@ -322,13 +324,70 @@ For now the JSON, YAML, and JS, are all being maintained, and there are no curre
 
 ## Aliases
 
-Targeting certain parts of an OpenAPI spec is powerful but it can become cumbersome to write and repeat complex JSON path expressions across various rules. Define aliases for commonly used JSON paths on a global level which can then be reused across the ruleset.
+Targeting certain parts of an OpenAPI spec is powerful but it can become cumbersome to write and repeat complex JSONPath expressions across various rules.
+Define aliases for commonly used JSONPath expressions on a global level which can then be reused across the ruleset.
 
 Aliases can be defined in an array of key-value pairs at the root level of the ruleset.
-
-```
-aliases: {
-  {Name}: '{JSONPath}'
+This is the JSON Schema definition describing aliases.
+It's a superset of `given`, with the notable difference being the possibility to distinguish between different formats.
+
+```json
```

Comment:
I'm not sure this will really be all that helpful for folks reading the docs, and might just be confusing? I'm pretty well versed with json schema, and even so it's not easy to imagine what this schema is trying to describe. The examples are much more helpful 🤷‍♂️

Reply:
True, I might just delete it. 
Do you think the examples here are sufficient or should I add a few additional ones?

Reply:
I think delete and good for now